lib/node_modules/@stdlib/blas/ext/base/sapxsum/docs/repl.txt

```diff
@@ -85,14 +82,13 @@
     Examples
     --------
     // Standard Usage:
-    > var x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 2.0 ] );
+    > x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 2.0 ] );
```

Comment:
We need to declare this, no?

Reply:
I had a look [here](https://github.com/stdlib-js/stdlib/pull/1741#discussion_r1518810886). This was the same case.

Reply:
I understood it, we declare it only for the first example. I have changed it now as per your suggestions. Thanks for pointing this out !